core/src/main/java/org/frankframework/http/AbstractHttpSession.java

```diff
@@ -180,8 +175,8 @@ public abstract class AbstractHttpSession implements ConfigurableLifecycle, HasK
 	private @Getter int maxExecuteRetries = 1;
 	private @Getter boolean staleChecking=true;
 	private @Getter int staleTimeout = 5_000; // [ms]
-	private @Getter int connectionTimeToLive = 900; // [s]
-	private @Getter int connectionIdleTimeout = 10; // [s]
+	private @Getter int connectionTimeToLive = 900; // [seconds]
+	private @Getter int connectionIdleTimeout = 10; // [seconds]
```

Comment:
Beter zo!
Ik denk dat het een goed idee is om de Frank!Doc bij de setters ook te updaten om dit te verduidelijken.